demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -87,10 +90,13 @@ def is_valid_file(self, validate_rn: bool = True, skip_test_conf: bool = False)
             self.is_valid_description(beta_integration=False),
         ]
 
+        answer3 = []
+        if self.support_level in LEVEL_SUPPORT_OPTIONS_3:
+            answers3 = [self.is_valid_max_fetch_and_first_fetch()]
```

Comment:
```suggestion
        answer3 = []
        if self.support_level in LEVEL_SUPPORT_OPTIONS_3:
            answers = answers.append(self.is_valid_max_fetch_and_first_fetch())
```

Reply:
I thought to keep the current structure so it will be more understood that there are support levels and matching "answers".

Reply:
I see, maybe we can rename with the other validations PR? it seems (to me) less clear this way now. @roysagi @amshamah419 WDYT.

Reply:
I'm in agreeance with @bakatzir. It's not really clear, but we could do something like `LEVEL_3_SUPPORT_ANSWERS` or something similar.